low-level/framework/messaging/ingress_processor.py

```diff
@@ -243,9 +257,7 @@ def _read_config(self):
         self._consumer_id = Conf.get(SSPL_CONF,
                                      f"{self.PROCESSOR}>{self.CONSUMER_ID}",
                                      'sspl_actuator')
-        self._consumer_group = Conf.get(SSPL_CONF,
-                                        f"{self.PROCESSOR}>{self.CONSUMER_GROUP}",
-                                        'cortx_monitor')
+        self._consumer_group = socket.getfqdn()
```

Comment:
Broadcasting requires all node to be in different consumer groups. As of now I am using fqdn. Please suggest if something else should be used or it should be extracted from config. @cdeshmukh @sandeepsgt @shriya-deshmukh

Reply:
cortx_monitor_<node_id>

we used to have this for rabbitmq routing key, which served similar purpose. Biding messages to node
So, I think you can use above

Reply:
I initially tried with node_id but we dont have node_id defined in SSPL_CONF so it takes the default value as `SN01`
```
self._node_id = Conf.get(SSPL_CONF,f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}", 'SN01')
```

Reply:
node_id is in global conf, not in sspl conf

Reply:
I have updated `consumer_group` to `consumer_group_prefix` in `sspl.conf` which will be prefixed with Node ID.